components/form/form-element-mixin.js

```diff
@@ -96,7 +96,7 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 	}
 
 	get label() {
-		return null;
+		return this.label ? this.label : null;
```

Comment:
So the lit analyzer was giving me the error
```
 This attribute binds the type 'null' which can be 'null'.
    173:  label="${ifDefined(this.label)}"
    no-nullable-attribute-binding
```

This change fixes it but I'm not sure if this is correct/causes problems with form element mixin. I did try different combinations of `label="${ifDefined(this.label === null ? undefined : this.label)"` in input-date and still had problems.

Reply:
Looks like `ifDefined` only checked for `undefined` values, not `null` values. So @AllanKerr maybe we could change the `FormElementMixin` to return `undefined` values instead of `null` ones?

Reply:
That also does not fix things:
```
This attribute binds the type 'undefined' which can be 'undefined'.
    172:  label="${ifDefined(this.label)}"
    no-nullable-attribute-binding
```

It actually seems like something weird is going on. I tried things from https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin#no-nullable-attribute-binding and they don't fix it

Reply:
Yeah weird eh... it just says to wrap it in an `ifDefined` call, which you're doing.

> Binding undefined or null in an attribute binding will result in binding the string "undefined" or "null". Here you should probably wrap your expression in the "ifDefined" directive.

Reply:
The act of defining `label` as a property should mean the getter in `form-element-mixin` never gets called because it automatically gets overrided. Lit creates getters for properties that look like:
```
get() {
    return this[key];
}
```

Reply:
Unfortunately we can't self-reference the property inside the getter like this:.
```
return this.label ? this.label : null;
```
If we do and someone chooses not to override `get label` then we'll get an infinite loop when trying to fetch the label for the element.

Reply:
Interesting - so perhaps there is some sort of bug with the linter?